Update Rahti loadbalancer service instrutions (#2979)
CSCfi/csc-user-guide: docs/cloud/rahti/networking.md

    app: httpd
```

### Add firewall IP blocking to a LoadBalancer Service using NetworkPolicy

> It is possible to add firewall IP blocking to a `LoadBalancer` Service. This means that you can add an allowlist of IPs (`188.184.77.250`) and/or IP masks (`188.184.0.0/16`) that will be the only ones that will be able to access the service. This is added to using secure protocols and safe password practises, can be a good improvement in security.

Reviewer: ```suggestion
It is possible to add firewall IP blocking to a `LoadBalancer` Service. This means that you can add an allowlist of IPs (`188.184.77.250`) and/or IP masks (`188.184.0.0/16`) that will be the only ones that will be able to access the service. This, combined with secure protocols and safe password practices, can significantly improve security.
```

Previous sentences do not sound correct. I think this suggestion makes it more clear :)
